tests/MonoGame.Extended.Tests/MockGameWindow.cs

```diff
@@ -10,11 +10,14 @@ public class MockGameWindow : GameWindow
     {
         public override bool AllowUserResizing { get; set; }
         public override Rectangle ClientBounds { get; }
-        public override Point Position { get; set; }
         public override DisplayOrientation CurrentOrientation { get; }
         public override IntPtr Handle { get; }
         public override string ScreenDeviceName { get; }
 
+#if !FNA
+        public override Point Position { get; set; }
```

Comment:
GameWindow.Position is an MG extension that's wrapping ClientBounds. It doesn't exist in KNI/FNA.